flutter_lyra_android/test/flutter_lyra_android_test.dart

```diff
@@ -73,8 +73,8 @@ void main() {
 
       test('$PaymentCancelledByUserExceptionInterface', () {
         final exception = PlatformException(
-          code: 'code',
-          message: errorCodesInterface.paymentCancelledByUser,
+          code: errorCodesInterface.paymentCancelledByUser,
```

Comment:
Since we are doing a `.contain` in the real code

I'd like the test to show that we need this `contain` and that a `==` would not work
Could you make this exception closer to a real exception we would receive from the native side ?